src/pages/Home/Home.test.js

```diff
@@ -0,0 +1,338 @@
+    expect(emptyWord).toHaveLength(1);
+
+    const emptyLetters = screen.getAllByRole('button', { name: 'empty' });
+    expect(emptyLetters).toHaveLength(5);
+  });
+});
+
+test('enter all incorrect letters (CLASS)', async () => {
+  // get all the keyboard buttons we will use
+  const CButton = getKeyboardLetter('C');
+  const LButton = getKeyboardLetter('L');
+  const AButton = getKeyboardLetter('A');
+  const SButton = getKeyboardLetter('S');
+
+  // type CLASS with the keyboard buttons
+  fireEvent.click(CButton);
+  fireEvent.click(LButton);
+  fireEvent.click(AButton);
+  fireEvent.click(SButton);
+  fireEvent.click(SButton);
+
+  // check there are no more empty spaces and now it has CLASS content
+  const emptyWord = screen.queryByRole('group', { name: 'empty' });
+  expect(emptyWord).not.toBeInTheDocument();
+
+  const classWord = screen.queryByRole('group', { name: 'CLASS' });
+  expect(classWord).toBeInTheDocument();
+
+  const emptyLetters = screen.queryByRole('button', { name: 'empty' });
+  expect(emptyLetters).not.toBeInTheDocument();
+
+  const CLetters = screen.getAllByRole('button', { name: 'C' });
+  expect(getLetters(CLetters)).toHaveLength(1);
+
+  const LLetters = screen.getAllByRole('button', { name: 'L' });
+  expect(getLetters(LLetters)).toHaveLength(1);
+
+  const ALetters = screen.getAllByRole('button', { name: 'A' });
+  expect(getLetters(ALetters)).toHaveLength(1);
+
+  const SLetters = screen.getAllByRole('button', { name: 'S' });
+  expect(getLetters(SLetters)).toHaveLength(2);
+
+  // submit the word
+  const enterButton = screen.getByRole('button', { name: /enter/i });
+  fireEvent.click(enterButton);
+
+  await waitFor(() => {
+    // the CLASS letter's row and CLAS keys has all absent aria label (9 in total)
+    const absentLetters = screen.getAllByRole('button', { name: /absent/i });
+    expect(absentLetters).toHaveLength(9);
+
+    // there is a new row for new empty word
+    const newLineEmptyWord = screen.getByRole('group', { name: 'empty' });
+    expect(newLineEmptyWord).toBeInTheDocument();
+
+    const newLineEmptyLetters = screen.getAllByRole('button', { name: 'empty' });
+    expect(newLineEmptyLetters).toHaveLength(5);
+  });
+});
+
+test('enter some misplaced letters (BLEED)', async () => {
+  // get all the keyboard buttons we will use
+  const BButton = getKeyboardLetter('B');
+  const LButton = getKeyboardLetter('L');
+  const EButton = getKeyboardLetter('E');
+  const DButton = getKeyboardLetter('D');
+
+  // type CLASS with the keyboard buttons
+  fireEvent.click(BButton);
+  fireEvent.click(LButton);
+  fireEvent.click(EButton);
+  fireEvent.click(EButton);
+  fireEvent.click(DButton);
+
+  // check there are no more empty spaces and now it has BLEED content
+  const emptyWord = screen.queryByRole('group', { name: 'empty' });
+  expect(emptyWord).not.toBeInTheDocument();
+
+  const bleedWord = screen.queryByRole('group', { name: 'BLEED' });
```

Comment:
```suggestion
  const bleedWord = screen.getByRole('group', { name: 'BLEED' });
```
also BLEED can be on a constant